### PR TITLE
chore(deployer): redirect moved pages via lambda

### PR DESCRIPTION
**Pull request by caugner**
### Problem

Currently, we redirect moved pages by adding corresponding objects in S3. This means we currently have >35'000 files just for redirects.

### Solution

Handle those redirects in the `content-origin-request` lambda, by merging the `_redirects.txt` files from mdn/content and mdn/translated-content into a `redirects.json` file containing a flat object mapping of target path by lowercased source path.

---

## How did you test this change?

Manually packaged (`yarn run make-package`) and uploaded the lambda on AWS and tested with test events.

### CODE REVIEW

File: deployer/aws-lambda/content-origin-request/index.js [hunk 1]
```diff
@@ -136,6 +138,11 @@ exports.handler = async (event) => {
     return redirect(`/${locale}${path || "/"}` + qs);
   }
 
+  // Redirect moved pages (see `_redirects.txt` in content/translated-content).
+  if (typeof REDIRECTS[requestURILowerCase] == "string") {
+    return redirect(REDIRECTS[requestURILowerCase]);
+  }
```
Comment (caugner) [inline deployer/aws-lambda/content-origin-request/index.js:144]: Hm, this only redirects HTML requests, but the JSON requests (`index.json`) should probably be redirected as well. 🤔

File: deployer/aws-lambda/content-origin-request/build.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
+/* eslint-disable node/no-unpublished-require */
```
Comment (caugner) [inline deployer/aws-lambda/content-origin-request/build.js:1]: Later we could reuse this mapping to support these redirects with MDN Offline. 🤔

File: deployer/aws-lambda/content-origin-request/index.js (hunk 1 above)
Comment (caugner) [reply deployer/aws-lambda/content-origin-request/index.js]: ✅ in f91e2beb1c38fed43a7c255e02484399fe5f2bd2.

**Comment by caugner** [comment]
@fiji-flo cb99267582a427e7b516c48cf5a26d40c046b356 is now deployed on stage.

**Comment by caugner** [comment]
✅ Rebased.

**Review by fiji-flo — APPROVED** [review]
Great.

**Pull request merged**